README.md

```diff
@@ -1,2 +1,28 @@
-This is a wrapper for Docker Credential helpers created by Azure Container Registry (ACR) team. This tool allows Azure Active Directory (AAD) based login. The credentials will be saved by the underlying cred-helpers configured at build time.
+# ACR Docker login helper
 
+This is a wrapper for Docker Credential helpers created by Azure Container Registry (ACR) team. This credential helper make use of Azure Active Directory (AAD) to obtain and maintain user's credentials.
+
+## Prerequisites
+
+- Docker installation is required, of course.
+- [Azure CLI](https://github.com/Azure/azure-cli)
+
+## Installation
+For windows run the [powershell installation script](https://aka.ms/acr/installaad/win) in administrator mode:
+
+`iex ([System.Text.Encoding]::ASCII.GetString((Invoke-WebRequest -Uri https://aka.ms/acr/installaad/win).Content))`
+
+For linux, run the [bash installation script](https://aka.ms/acr/installaad/bash) as root:
+
+`curl -L https://aka.ms/acr/installaad/bash | sudo /bin/bash`
+
+## Usage
+To login to an ACR service as followed:
+    `az acr login -n <registry name>`
+
+After that, you will be able to use docker normally. This credential helper would help maintaining your AAD access tokens.
```

Comment:
Change the last part to: This credential helper will help maintaining your credentials valid.